src/components/productListing/ProductCard.js

```diff
@@ -12,10 +12,13 @@ const ProductCard = ({ product }) => {
   const navigate = useNavigate();
   const { cart, setCart } = useCart();
   const { wishList, setWishList } = useWishlist();
+
   return (
     <div className="product-card" key={product.id}>
       <div className="product-img-div">
-        <img src={product.image} alt={`${product.name} image`} />
+        <Link to={`/product/${product._id}`}>
+          <img src={product.image} alt={`${product.name} image`} />
```

Comment:
in the `alt` attribute you shouldn't use the work image/picture/photo etc.

Reply:
okay , I will do the changes